core/templates/core/user_tools.jinja

```diff
@@ -42,152 +42,190 @@
+                  <li class="rows">
+                    <strong>{% trans %}Club account: {% endtrans %}</strong>
+                    <a href="{{ url('accounting:club_details', c_account_id=ca.id) }}">{{ ca }}</a>
+                  </li>
+                {%- endfor %}
+              {%- endif -%}
+            {%- endfor %}
+          </ul>
+        </div>
+      {% endif %}
+
+      {% if user.is_root or user.is_com_admin or user.is_in_group(pk=settings.SITH_GROUP_SAS_ADMIN_ID) %}
+        <div>
+          <h4>{% trans %}Communication{% endtrans %}</h4>
+          <ul>
+            {% if user.is_com_admin or user.is_root %}
+              <li><a href="{{ url('com:weekmail_article') }}">{% trans %}Create weekmail article{% endtrans %}</a></li>
+              <li><a href="{{ url('com:weekmail') }}">{% trans %}Weekmail{% endtrans %}</a></li>
+              <li><a href="{{ url('com:weekmail_destinations') }}">{% trans %}Weekmail destinations{% endtrans %}</a></li>
+              <li><a href="{{ url('com:news_new') }}">{% trans %}Create news{% endtrans %}</a></li>
+              <li><a href="{{ url('com:news_admin_list') }}">{% trans %}Moderate news{% endtrans %}</a></li>
+              <li><a href="{{ url('com:alert_edit') }}">{% trans %}Edit alert message{% endtrans %}</a></li>
+              <li><a href="{{ url('com:info_edit') }}">{% trans %}Edit information message{% endtrans %}</a></li>
+              <li><a href="{{ url('core:file_moderation') }}">{% trans %}Moderate files{% endtrans %}</a></li>
+              <li><a href="{{ url('com:mailing_admin') }}">{% trans %}Mailing lists administration{% endtrans %}</a></li>
+              <li><a href="{{ url('com:poster_list') }}">{% trans %}Posters{% endtrans %}</a></li>
+              <li><a href="{{ url('com:screen_list') }}">{% trans %}Screens{% endtrans %}</a></li>
+            {% endif %}
+            {% if user.is_in_group(pk=settings.SITH_GROUP_SAS_ADMIN_ID) %}
+              <li><a href="{{ url('sas:moderation') }}">{% trans %}Moderate pictures{% endtrans %}</a></li>
+            {% endif %}
+          </ul>
+        </div>
+      {% endif %}
+
+      {% if user.has_perm("subscription.add_subscription") or user.has_perm("auth.change_perm") or user.is_root or user.is_board_member %}
+        <div>
+          <h4>{% trans %}Subscriptions{% endtrans %}</h4>
+          <ul>
+            {% if user.has_perm("subscription.add_subscription") %}
+              <li>
+                <a href="{{ url("subscription:subscription") }}">
+                  {% trans %}New subscription{% endtrans %}
+                </a>
+              </li>
+            {% endif %}
+            {% if user.has_perm("auth.change_permission") %}
+              <li>
+                <a href="{{ url("subscription:perms") }}">
+                  {% trans %}Manage permissions{% endtrans %}
+                </a>
+              </li>
+            {% endif %}
+            {% if user.is_root or user.is_board_member %}
```

Comment:
Ça devrais pas être une autre permission ça ? Genre view_subscription_stats ou un truc du genre ?

Reply:
créer une permission, ça demanderait une migration

Reply:
Out of scope selon toi ?